torch/export/_trace.py

```diff
@@ -439,6 +356,11 @@ def _export_non_strict(
             trace_joint=False,
             pre_dispatch=pre_dispatch,
         )
+    # TODO unfortunately preserving graph-level metadata is not
```

Comment:
🤔 in theory the stuff Sherlock added for preserving nn stack traces from dynamo also means that node meta should be preserved

Reply:
Oh, graph level meta